medusa/indexers/tmdb/tmdb.py

```diff
@@ -182,11 +182,27 @@ def search(self, series):
         """
         log.debug('Searching for show: {0}', series)
 
-        results = self._show_search(series, request_language=self.config['language'])
-        if not results:
+        results = None
+        # If search term is digit's only, store it and attempt to search by id.
+        show_by_id = None
+
+        try:
+            if series.isdigit():
+                show_by_id = self._get_show_by_id(series, request_language=self.config['language'])
+            results = self._show_search(series, request_language=self.config['language'])
```

Comment:
This will trigger a double search. Intentional?

Reply:
Yes. If a show name has only numbers, it can still be found. Don't know if they exist, but this solution is prepared for that